sarathi/model_executor/models/llama.py

```diff
@@ -171,12 +172,13 @@ def forward(
         positions: torch.Tensor,
         hidden_states: torch.Tensor,
         kv_cache: KVCache,
```

Comment:
now that kv cache is managed by the wrapper itself, we don't need to pass both kv_cache and the wrapper

Reply:
During the initial profiling (to determine how many cache blocks to allocate to the model) of the model, the we pass an empty cache tensor to the attention_backend. Due to this, I haven't removed the parameter.

Reply:
Made the changes